packages/lego-bricks/src/components/Button/Button.module.css

```diff
@@ -62,22 +71,14 @@
   > a {
```

Comment:
The `+ .button` rule above here causes issues with vertical lists of Buttons/LinkButtons, like the event admin panel